macro/derive/src/lib.rs

```diff
@@ -8,20 +8,26 @@ use quote::quote;
 /// Attribute for skipping from attribute
 const SKIP_FROM_ATTR: &str = "skip_from";
 const SKIP_TRY_FROM_ATTR: &str = "skip_try_from";
+/// Attribute to skip inner container optimization. Useful for trait objects
```

Comment:
In the future, these sorts of small changes can be merged separately. It can help with the reviewing process.

Reply:
Okay, understood